propriedade/tests/views/test_propriedade.py

```diff
@@ -219,27 +219,81 @@ class PropriedadeHistoricoPlantioAPIView(APITestMixin, TestCase):
 
     def setUp(self):
         self.propriedade = propriedade.make()
-        self.url = reverse_lazy(
-            "propriedade-historico-plantio", kwargs={'idPropriedade': self.propriedade.idPropriedade})
+        self.url = self.get_url()
         self.talhoes = talhao.make(idPropriedade=self.propriedade, _quantity=3)
         talhao.make(idPropriedade=self.propriedade, _quantity=2)
         for t in self.talhoes:
             plantio.make(talhao=t)
 
+    def get_url(self, idPropriedade=None):
+        if idPropriedade is None:
+            idPropriedade = self.propriedade.idPropriedade
+        return reverse_lazy("propriedade-historico-plantio", kwargs={'idPropriedade': idPropriedade})
+
     def test_lista_plantios_do_talhao(self):
         response = self.client.get(self.url)
         self.assertEqual(response.status_code, 200, response.json())
         self.assertEqual(len(response.json()), 3)
 
     def test_nao_lista_plantio_de_outra_propriedade(self):
-        url = reverse_lazy(
-            "propriedade-historico-plantio", kwargs={'idPropriedade': self.propriedade.idPropriedade})
         propriedade_diferente = propriedade.make()
         talhoes = talhao.make(idPropriedade=propriedade_diferente, _quantity=3)
         for t in talhoes:
             plantio.make(talhao=t)
 
-        response = self.client.get(url)
+        response = self.client.get(self.url)
         self.assertEqual(response.status_code, 200, response.json())
         self.assertEqual(Plantio.objects.count(), 6)
         self.assertEqual(len(response.json()), 3)
+
+
+class PropriedadeDeleteTecnicoAPIView(APITestMixin, TestCase):
+
+    def setUp(self):
+        self.produtor = produtor.make()
+        self.tecnico = tecnico.make()
+        self.propriedade = propriedade.make(tecnico=self.tecnico)
+        self.url = self.get_url()
+
+    def get_url(self, idPropriedade=None):
+        if idPropriedade is None:
+            idPropriedade = self.propriedade.idPropriedade
+        return reverse_lazy("propriedade-delete-tecnico", kwargs={'idPropriedade': idPropriedade})
+
+    def test_deleta_tecnico_propriedade(self):
+        self.client.credentials(HTTP_AUTHORIZATION='Bearer ' + self.get_header_credencial(self.tecnico.usuario))
+
+        response = self.client.delete(self.url)
+        data = Propriedade.objects.filter(idPropriedade=self.propriedade.idPropriedade)[0]
```

Comment:
Só passando como comentário, aqui é melhor usar o first
```suggestion
        data = Propriedade.objects.filter(idPropriedade=self.propriedade.idPropriedade).first()
```